src/python/grpcio/grpc/__init__.py

```diff
@@ -299,6 +299,27 @@ class RpcError(Exception):
     """Raised by the gRPC library to indicate non-OK-status RPC termination."""
 
 
+class AioRpcError(RpcError):
```

Comment:
If this goes directly into `__init__.py`, it signals inclusion into our stable public API. Can we somehow make this live under the `experimental` module?